package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -552,16 +561,57 @@ def _read_next_timestep(self):
 
         indices = np.zeros(self.n_atoms, dtype=int)
 
-        f.readline()  # ITEM ATOMS etc
-        for i in range(self.n_atoms):
-            idx, _, xs, ys, zs = f.readline().split()
+        atomline = f.readline()  # ITEM ATOMS etc
+        attrs = atomline.split()[2:]  # attributes on coordinate line
+        col_ids = {}  # column index of each attribute
+        for i, attr in enumerate(attrs):
+            col_ids[attr] = i
+
+        # check for ids and what type of coordinate convention
+        ids = "id" in col_ids.keys()
+        # keys = convention values = col_ids or False if not present
+        coord_dict = {"unscaled": False, "scaled": False, "unwrapped": False,
+                      "scaled_unwrapped":False}
+        if "x" and "y" and "z" in col_ids.keys():  # unscaled
```

Comment:
Won't this be doing the Boolean logic on the left before the "in" check? I think you need something gross like "x in col AND y in col"

Reply:
Doh...